### PR TITLE
PS-1: Week-7: Item and IndentFile2 table has been added in database

### PR DESCRIPTION
**Pull request by Shiv2708**
## Proposed changes
- Item and IndentFile2 table has been added in database to support multiple items in same indent
- previous table has not been changed so there will be no issue while migrating database

### Brief description of what is fixed or changed
- In previous versions the database did not support multiple items in same indent file so new table was added to support this feature. 
- Previous table is not removed so to avoid issues with all other endpoints

## Types of changes

_Put an `x` in the boxes that apply_

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Other (please describe):

## Checklist

_Put an `x` in the boxes that apply_

-   [x] My code follows the [style guidelines of this project](https://docs.google.com/document/d/1GI2Hile8UbGZ82gFe1y4wAVPcNPXip1cmXTzog1S41U/edit)
-   [x] I have performed a self-review of my own code
-   [x] I have created new branch for this pull request
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [x] My changes generate no new warnings
-   [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots

db_schema

### CODE REVIEW

**Comment by Shiv2708** [comment]
@PranshuNayak 
@samay-rgb

File: FusionIIIT/applications/ps1/models.py [hunk 1]
```diff
@@ -27,6 +27,47 @@ class IndentFile(models.Model):
...
+    procured = models.BooleanField(default=False)
+    description = models.CharField(max_length=250,blank=False,default="")
+
+    class Meta:
+        db_table = 'IndentFile2'
+
+
+class Item(models.Model):
+    item_id = models.AutoField(primary_key=True)
+    indent_id = models.ForeignKey(
```
Comment (PranshuNayak) [inline FusionIIIT/applications/ps1/models.py]: Change the field_name to 'indent_file_id'

File: FusionIIIT/applications/ps1/models.py [hunk 2]
```diff
@@ -27,6 +27,47 @@ class IndentFile(models.Model):
     class Meta:
         db_table = 'IndentFile'
 
+
+class IndentFile2(models.Model):
+
+    file_info = models.OneToOneField(
+        File, on_delete=models.CASCADE, primary_key=True)
+    indent_title=models.CharField(max_length=250,blank=False,default="")
```
Comment (samay-rgb) [inline FusionIIIT/applications/ps1/models.py]: Use title instead of indent_title

**Comment by PranshuNayak** [comment]
@Shiv2708 push the migrations file too for this new model schemas

**Comment by kanha638** [comment]
@samay-rgb  changed the field name as mentioned.

**Comment by kanha638** [comment]
@PranshuNayak  migrations files can not be pushed as they are in .gitignore

**Comment by PranshuNayak** [comment]
@kanha638  Could you pull the update from upstream:ps1, you can push the migration files.

**Comment by kanha638** [comment]
@PranshuNayak pushed migration files

**Comment by PranshuNayak** [comment]
LGTM!

**Review by PranshuNayak — APPROVED** [review]
LGTM! Just squash the commits

**Comment by kanha638** [comment]
> LGTM! Just squash the commits

Done 👍🏻

**Comment by kanha638** [comment]
@PranshuNayak done

**Comment by PranshuNayak** [comment]
@kanha638 you have mistakenly pushed other files not related to this PR too.

**Comment by kanha638** [comment]
@PranshuNayak resolved. Squashing will create that issue as i have rebased this branch from ps-1 with changes so it also consider them on this PR if i squash commits.

**Review by PranshuNayak — APPROVED** [review]
approved @akshatnema 
@yatharthagoenka kindly deploy

**Pull request merged**